app/models/ballot_item.rb

```diff
@@ -3,4 +3,19 @@ class BallotItem < ActiveRecord::Base
 belongs_to :award
 belongs_to :play
 
+# play_id == convergence of play title and theater
+  def approved_ballot
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)